notSoGoodParts/this/README.md

```diff
@@ -0,0 +1,26 @@
+# Should We Use This ?
+
+Although ```this``` is not a new feature of es6 Douglas Crockford discussed the negatives of using ```this``` in [The Better Parts](https://www.youtube.com/watch?v=PSGEjv3Tqo0).
```

Comment:
when you include code in-line with text you only need to use a single backtick character around the `code snipit`. :wink: